### PR TITLE
New novelty mechanisms in PVLV and BOA: LDT instead of PPTg, SC novelty

### PR DESCRIPTION
**Pull request by rcoreilly**
Now using the first pool in BLAPos Acq and Ext for novelty, which is included by default and incorporated into PVLV etc.

### CODE REVIEW

File: chans/nmda.go [hunk 1]
```diff
@@ -47,7 +47,7 @@ func (np *NMDAParams) MgGFmVbio(vbio float32) float32 {
 	if vbio >= 0 {
 		return 0
 	}
-	return 1.0 / (1.0 + np.MgFact*mat32.FastExp(-0.062*vbio))
+	return -vbio / (1.0 + np.MgFact*mat32.FastExp(-0.062*vbio))
```
Comment (siboehm) [inline chans/nmda.go:50]: Can you add a comment about where this formula comes from?

File: examples/boa/boa.go [hunk 2]
```diff
@@ -1089,11 +978,12 @@ func (ss *Sim) Log(mode etime.Modes, time etime.Times) {
 		// 	ss.Loops.Stop(etime.Trial)
 		// }
 
-		// trnEpc := ss.Loops.Stacks[etime.Train].Loops[etime.Epoch].Counter.Cur
-		// if ss.StopOnErr && trnEpc > 5 && ev.Rew < 0 {
-		// 	fmt.Printf("STOPPED due to negative reward for US: %d  %g\n", ev.US, ev.Rew)
-		// 	ss.Loops.Stop(etime.Trial)
-		// }
+		ev := TheSim.Envs[etime.Train.String()].(*Approach)
```
Comment (etuleu) [inline examples/boa/boa.go:981]: comment this out before committing?

**Comment by siboehm** [comment]
Do you want us to review this now or wait?

**Comment by rcoreilly** [comment]
you can start reviewing. I'll need to update all the tests with new params etc, so a few more things going in, but most of the rest is done.

File: examples/boa/boa.go (hunk 2 above)
Comment (rcoreilly) [reply examples/boa/boa.go:981]: it is harmless actually unless you hit the StopOnErr button and this is as good an error to check as any at this point..

**Comment by rcoreilly** [comment]
Ok this pr should be ready to go without the params fixes.  I just learned how to use git rebase -i so that was fun!

**Review by etuleu — APPROVED** [review]

**Pull request merged**